tests/smoke/service.bats

```diff
@@ -2,16 +2,17 @@
 
 @test "create" {
   l0 environment create env_name
-  l0 deploy create ./common/Service.Dockerrun.aws.json dpl_name1
-  l0 deploy create ./common/Service.Dockerrun.aws.json dpl_name2
+  l0 deploy create ./common/Service_stateful.Dockerrun.aws.json dpl_name_stateful1
+  l0 deploy create ./common/Service_stateful.Dockerrun.aws.json dpl_name_stateful2
+  l0 deploy create ./common/Service_stateless.Dockerrun.aws.json dpl_name_stateless
   l0 loadbalancer create --port 80:80/http env_name lb_name1
-  l0 service create env_name svc_name1 dpl_name1
-  l0 service create --loadbalancer lb_name1 --scale 2 env_name svc_name2 dpl_name1
+  l0 service create --stateful --loadbalancer lb_name1 --scale 2 env_name svc_name_stateful dpl_name_stateful1
+  l0 service create env_name svc_name_stateless dpl_name_stateless
```

Comment:
Recommend adding the `:latest` suffix as having more than one version of this deploy will lead to an error.

Reply:
✅